djangocms_frontend/contrib/link/forms.py

```diff
@@ -31,6 +30,17 @@
 from .constants import LINK_CHOICES, LINK_SIZE_CHOICES, TARGET_CHOICES
 from .helpers import get_choices, get_object_for_value
 
+# Weak dependency on djangocms_icon
+# (Even if djangocms_icon is in the python path, the admin form will fail due to missing
+# templates if it's not in INSTALLED_APPS)
+if "djangocms_icon" in settings.settings.INSTALLED_APPS:
```

Comment:
Settings is duplicated here. Reviewing on my phone so assuming it's an error.

Reply:
You caught me there. Correct but bad style. settings.settings is the indirect referral to django.conf.settings Apparently that can be made more transparent. 👌

Reply:
@fsbraun Right I've had a look through the code & I get it now. I think this is confusing though, so would be better to change it to be clear what's happening there.

In the app's `settings.py` I'd probably do a `settings as dj_settings`. Then this file is already importing django settings;

```
from django.conf import settings as django_settings
```

So just use that directly rather than via the other settings file.

Reply:
Exactly. I just pushed the change.